packages/mgt-components/src/components/mgt-person-card/mgt-person-card.graph.ts

```diff
@@ -120,7 +120,7 @@ function buildWorksWithRequest(batch: IBatch, userId: string) {
 
 // tslint:disable-next-line:completed-docs
 function buildMessagesWithUserRequest(batch: IBatch, emailAddress: string) {
-  batch.get(batchKeys.messages, `me/messages?$search="from:${emailAddress}"`, ['Mail.ReadBasic']);
+  batch.get(batchKeys.messages, `me/messages?$search="from:${emailAddress}"`, ['Mail.Read']);
```

Comment:
Any reason why we're increasing the permission requested here?

Reply:
It was for an added feature for quick messaging on person-card but we're not currently moving forward with it. Removing.